scripts/lst-rfperformance.py

```diff
@@ -0,0 +1,114 @@
+"""Pipeline for test the performance of Random Forests.
+
+Usage:
+
+$>python lst-rfperformance.py
+
+"""
+import sys                                                   
+sys.path.insert(0, '../')
+import numpy as np
+import pandas as pd
+import argparse
+import matplotlib.pyplot as plt 
+from sklearn.externals import joblib
+import sys
+sys.path.insert(0, '../')
```

Comment:
This line can be removed since you import lstchain

Reply:
I get an error "No module named 'lstchain'" if I don't include the line :s

Reply:
Did you install it ?

Reply:
Yep, now it's working, sorry! We can do a cleaning of the code because I'm sure there are other parts where that line remains. Thank you